src/stories/button/Button.style.js

```diff
@@ -0,0 +1,45 @@
+import styled, { css } from 'styled-components';
+import theme from '../../theme';
+
+export const StyledButtonComponent = styled.button`
+  text-transform: uppercase;
+  border: 2px solid ${theme.white};
+  font-style: italic;
+
+  font-size: 1rem;
+  font-weight: 600;
+  background-color: ${({ type }) =>
+    type === 'secondary' ? theme.basiccolor : theme?.[type]};
```

Comment:
@Arbina001 can you please explain this code

Reply:
this is white color which i have used in my theme